lib/sisense/api/client.rb

```diff
@@ -63,8 +70,9 @@ def request(method, path, params)
       end
 
       def encode_path(path, params = nil)
```

Comment:
**params here too... unless it's a single variable then give it a different name since params is conventionally used for the collection of additional named parameters as hash keys.